### ISSUE TITLE
Include Load Data from Ops in Influx Events

### ISSUE THREAD

**Issue by AlecHFerguson** [creditkarma/good-influx#4]
There is some information content in Ops data events that is not included in the events sent to Influx by this module.  We would find it helpful in monitoring our Hapi services, so we propose including it.  I would actually be happy to write the code and submit a pull request.

In particular, `load.requests`, `load.concurrents`, `load.responseTimes` and `load.sockets` are not included in the `line-protocol` formatter.  I suspect it was done this way because they are on a per-port basis _(See example event pasted below)_.  How should these events be flattened?  Here are a few proposals -- What does the community think?

### 1. Separate Events into Two or More
The first event sent would be the information that is sent now.  Then, a `load` event would be sent for each port.  It would be flattened from the statistics in `load`:

```
ops,host=mytesthost,pid=9876 port=8080,requestsTotal=50,requestsDisconnects=0,requests200=50,concurrents=3... 123456789000000
ops,host=mytesthost,pid=9876 port=9876,requestsTotal=63,requestsDisconnects=1,requests200=61,concurrents=4... 123456789000000
```
Each load event would also be named "ops"; and would also have host, pid and timestamp.  This would mean multiple events in InfluxDB per `ops` event emitted from `good`.  Though they should still be searchable.

### 2. Include the load data with a port name
Fields would be named such that they contain the port.  I.E., if the port is 8080 then the Influx event would have these fields:

```
requestsTotal8080=50,requestsDisconnects8080=0,requests200_8080=50,concurrents8080=3
```

I.E., the port would be appended to the naming.  This would maintain one row in InfluxDB per `ops` event emitted by `good`.  My concern with this approach is that the names of fields are inconsistent.

### 3. Average the values for multiple ports
This would include just one InfluxDB event per ops event.  Though it might mask important issues.  Like if port 8080 response times spiked, but port 1234 went down and response times dropped significantly, you might not know it.

I would prefer approach **1**.  Would be good to get some feedback.

### Example event
```
{
  "event": "ops",
  "timestamp": 1485996802647,
  "host": "myservice.awesome.com",
  "pid": 128,
  "os": {
    "load": [
      3.05078125,
      2.11279296875,
      1.625
    ],
    "mem": {
      "total": 6089818112,
      "free": 147881984
    },
    "uptime": 23489
  },
  "proc": {
    "uptime": 22.878,
    "mem": {
      "rss": 64290816,
      "heapTotal": 47271936,
      "heapUsed": 26825384
    },
    "delay": 32.292509999126196
  },
  "load": {
    "requests": {
      "8080": {
        "total": 1,
        "disconnects": 0,
        "statusCodes": {
          "200": 1
        }
      }
    },
    "concurrents": {
      "8080": 0
    },
    "responseTimes": {
      "8080": {
        "avg": 990,
        "max": 990
      }
    },
    "sockets": {
      "http": {
        "total": 0
      },
      "https": {
        "total": 0
      }
    }
  }
}
```

**Comment by fhemberger** [comment]
I like the first approach as well, but I would call the event `load` or `ops_load` instead of mixing it with the `ops`event name. This would also make the change backwards compatible.

**Comment by AlecHFerguson** [comment]
Ah, true. I guess the ops event would not change from what it currently is, no? Sounds good. I should have time to work on this in the near future.

**Comment by AlecHFerguson** [comment]
An update: My team has actually been working on this enhancement.  I've tested it extensively with a node server of mine and it seems to be working nicely.  We will hopefully be opening a pull request in the near future.

**Comment by fhemberger** [comment]
@AlecHFerguson Awesome! :tada:

**Comment by fhemberger** [comment]
Closed via #12.
